distribution/docs/src/main/resources/content/_tables/Wfs_v1_1_0_Connected_Source.adoc

```diff
@@ -0,0 +1,104 @@
+:title: WFS 1.1.0 Connected Source
+:id: Wfs_v1_1_0_Connected_Source
+:type: table
+:status: published
+:application: ${ddf-spatial}
+:summary: WFS 1.1.0 Connected Source.
+
+.[[Wfs_v1_1_0_Connected_Source]]WFS v1.1.0 Connected Source
```

Comment:
From the blueprint / metatype, it doesn't look like we are creating a connected source implementation

Reply:
That brings up a good question. Do we need a connected source? And how is a connected source different from a regular source?

Reply:
Not sure we need it immediately; we can always add it later after the WFS 1.1.0 functionality is finalized

Reply:
k